authorization-server/src/modules/app/app.module.ts

```diff
@@ -38,7 +38,7 @@ try {
               ? {
                   target: 'pino-pretty',
                   options: {
-                    colorize: true,
+                    colorize: !configService.get<string>('NO_COLOR'),
```

Comment:
wouldn't it be more readable if this var was named `COLORIZE`?

Reply:
This is kind of "standard" AFAIK. Standard NestJS logger respects this, but it is undocumented. So, I decided to keep this name.

Reply:
Ok then